include/libdnf/module/module_package.hpp

```diff
@@ -0,0 +1,222 @@
+/*
+Copyright Contributors to the libdnf project.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2.1 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#ifndef LIBDNF_MODULE_MODULE_PACKAGE_HPP
+#define LIBDNF_MODULE_MODULE_PACKAGE_HPP
+
+#include "libdnf/module/module_profile.hpp"
+
+#include <string>
+#include <vector>
+
+struct _ModulemdModuleStream;
+
+
+namespace libdnf::module {
+
+
+struct ModulePackageId {
+public:
+    ModulePackageId() = default;
```

Comment:
Why did you created a default constructor. I remember only one reason -SWIG when it is in vector. Is there any other reason?

Reply:
I did it the same as for e.g. comps::Group or rpm::Package. But I don't remember why it's needed.